src/system/ui/lists/style.scss

```diff
@@ -225,6 +225,11 @@
 		.fl-asst-is-favorite {
 			color: var(--fl-asst-bookmark-color);
 		}
+	}
+
+
+	.fl-asst-comment-extras {
+        display: block !important;
```

Comment:
These styles should probably be with the relevant component or inside the app if specific only to the app.

Reply:
Moved styles to comment app style file.